rpcs3/Emu/Cell/Modules/cellMsgDialog.cpp

```diff
@@ -124,6 +119,13 @@ s32 cellMsgDialogOpen2(u32 type, vm::cptr<char> msgString, vm::ptr<CellMsgDialog
 	return CELL_OK;
 }
 
+s32 cellMsgDialogOpen(u32 type, vm::cptr<char> msgString, vm::ptr<CellMsgDialogCallback> callback, vm::ptr<void> userData, vm::ptr<void> extParam)
+{
+	//Note: This function needs proper implementation, solve first argument "type" conflict with MsgDialogOpen2 in cellMsgDialog.h.
+	cellSysutil.todo("cellMsgDialogOpen(type=0x%x, msgString=%s, callback=*0x%x, userData=*0x%x, extParam=*0x%x)", type, msgString, callback, userData, extParam);
+	return	cellMsgDialogOpen2(type, msgString, callback, userData, extParam);
```

Comment:
@Nekotekina Problem with this change, now log indicate 3 time same line 

>·U {PPU[0x1000000] Thread (main_thread) [0x00611e70]} cellSysutil TODO: cellMsgDialogOpen(type=0x0, msgString=“This game supports the system data Auto Save function.
Do not power off the system when the HDD access indicator is flashing.”, callback=*0x642dc8, userData=*0x101d9720, extParam=*0x0)
·W {PPU[0x1000000] Thread (main_thread) [0x00611e70]} cellSysutil: cellMsgDialogOpen2(type=0x0, msgString=“This game supports the system data Auto Save function.
Do not power off the system when the HDD access indicator is flashing.”, callback=*0x642dc8, userData=*0x101d9720, extParam=*0x0)
·E {PPU[0x1000000] Thread (main_thread) [0x00611e70]} cellSysutil: This game supports the system data Auto Save function.
Do not power off the system when the HDD access indicator is flashing.

First have a 3 time line message for game 
>This game supports the system data Auto Save function.
Do not power off the system when the HDD access indicator is flashing.

And after have 2 time line message:
>type=0x0, msgString=“”, callback=*0x642dc8, userData=*0x101d9720, extParam=*0x0)